### PR TITLE
Creating an ExpressJS server for static assets

### PR DESCRIPTION
**Pull request by jicksy**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -0,0 +1,22 @@
+{
+  "name": "pouchdb-hoodie-store-server",
+  "version": "1.0.0",
+  "description": "> tiny server with a front-end store API",
```
Comment (gr2m) [inline package.json]: I'd remove the `> ` prefix of description

**Comment by gr2m** [comment]
`package.json` looks good, just see my comment above. Next up would be adding a `.gitignore` and then an `index.js` file that probably returns a function with a callback, which starts a static file server at `options.path`. And then a `./bin/pouchdb-hoodie-store-server` file which runs the function from `./index.js` with the current folder. Makes sense?

But I'd merge this first and then do the rest in other pull requests

**Comment by jicksy** [comment]
Greg,
I have removed > prefix of description and also added .gitignore

**Comment by gr2m** [comment]
what is the `package.json~` file? Can you remove this one from the pull request?

File: package.json [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/jicksy/pouchdb-hoodie-store-server"
+  },
+  "author": "",
+  "license": "ISC",
+  "bugs": {
+    "url": "https://github.com/jicksy/pouchdb-hoodie-store-server/issues"
+  },
+  "homepage": "https://github.com/jicksy/pouchdb-hoodie-store-server",
```
Comment (gr2m) [inline package.json]: set the github URLs to the main repo: `github.com/hoodiehq/...`

**Comment by gr2m** [comment]
Close, as https://github.com/hoodiehq/pouchdb-hoodie-store-server/pull/6 got merged :)

**Pull request closed without merging**